utils/cliutils/utils.go

```diff
@@ -52,6 +52,7 @@ func GetCliError(err error, success, failed int, failNoOp bool) error {
 type detailedSummaryRecord struct {
 	Source string `json:"source"`
 	Target string `json:"target"`
+	Sha256 string `json:"sha256,omitempty"`
```

Comment:
We'd prefer to print an empty sha256
```suggestion
	Sha256 string `json:"sha256"`
```